### PR TITLE
Gpudriver470

### PR DESCRIPTION
**Pull request by moes1**
1. Updating driver to 470.141.03 to fix an infoROM corruption issue. 
2. Updating cuda to 11.4.4 as 11.4.3 seems to have a conflict with the new driver. 
3. Updated nccl to 2.14.3-1 to bring nccl in line with the master branch.

### CODE REVIEW

File: ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nccl.sh [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+source /etc/profile.d/modules.sh
+module load mpi/hpcx
+git clone https://github.com/NVIDIA/nccl-tests.git
+pushd nccl-tests
+make MPI=1 MPI_HOME=${HPCX_MPI_DIR} CUDA_HOME=/usr/local/cuda
+popd
+mv nccl-tests /opt/.
+module unload mpi/hpcx
+
+# NCCL-Tests Preset Run Config
```
[the remark this answers is not in the document]
Comment (moes1) [reply ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nccl.sh:36]: There are probably other ways to handle this, but the ubuntu/common install_nvidiagpudriver and install_nccl scripts are installing cuda 11.6, which do not support the driver we want to use for the LTS image. See Table 3 in the link below: 

https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-major-component-versions

File: ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nvidiagpudriver.sh [hunk 2]
```diff
@@ -1,4 +1,40 @@
 #!/bin/bash
 set -ex
 
-$UBUNTU_COMMON_DIR/install_nvidiagpudriver.sh 1804 "b8db71bf86c4b282a96efcb56d6b1bc48df13117b1a46c3e457628fd401587cc"
+# Parameters
+RELEASE_VERSION=1804
+CHECKSUM="749183821ffc051e123f12ebdeb171b263d55b86f0dd7c8f23611db1802d6c37"
```
Comment (jeseszhang1010) [inline ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nvidiagpudriver.sh:6]: Where does this CHECKSUM get used?

Comment (moes1) [reply ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nvidiagpudriver.sh:6]: Ah yeah it looks like we don't need that. It also appears we don't use the CHECKSUM variable in our common install_nvidiagpudriver.sh script in the master branch anymore either. 

https://github.com/Azure/azhpc-images/blob/master/ubuntu/common/install_nvidiagpudriver.sh#L6

**Review by jithinjosepkl — APPROVED** [review]

File: ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nccl.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (moes1) [reply ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nccl.sh]: alright I see what you mean about the topology specialization. removed.

**Review by jeseszhang1010 — APPROVED** [review]

**Pull request merged**